src/decoder/wav.rs

```diff
@@ -94,9 +94,11 @@ where
                             value
                         })
                     } else {
-                        // > 32 bits we cannot handle, so we'll just return equilibrium
-                        // and let the iterator continue
-                        Some(Self::Item::EQUILIBRIUM)
+                        #[cfg(feature = "tracing")]
+                        tracing::error!("Unsupported WAV float bit depth: {}", bits);
+                        #[cfg(not(feature = "tracing"))]
+                        println!("Unsupported WAV float bit depth: {}", bits);
```

Comment:
we can do eprintln then it will print to stderr

Reply:
If we're settled on `tracing`, I would use only that. A simple `tracing` listener can print to stdout or stderr if necessary.
We can have an example on how to do that.

Reply:
I do not expect most users of rodio to know or use tracing (rodio is used by a lot of rust novices it seems). Since `tracing::error!` calls do nothing without tracing set up in the final binary cut out the (e)println would mean silent errors a lot of users.

We do want the tracing support since that provides a way of redirecting such errors (essential in a TUI app)

Reply:
What do you guys prefer? Let me know and I can wrap it up.

Reply:
In my opinion its ready as is, this behavior matches what we do when we run into cpal stream errors. If we refine that later we can take a look at this again. But that is a separate concern for a separate PR/Issue. @PetrGlad would you agree?

Reply:
Except I should probably still change to `eprintln`?

Reply:
oh yes that would be nice

Reply:
Yes we can keep the error reporting as is for now, since this is the existing pattern. 

I do not think users should know how to use tracing. They only need to know how to enable debug output from rodio. I will have an example for this.

Reply:
> oh yes that would be nice

Changed to `eprintln` in [9861056](https://github.com/RustAudio/rodio/pull/669/commits/9861056120d6b7ecc9826585d6a040802106a9fe).